go/pkg/clickhouse/migrations/20250924104123.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE `default`.`api_requests_raw_v2` ADD COLUMN `query_string` String;
+ALTER TABLE `default`.`api_requests_raw_v2` ADD COLUMN `query_params` Map(String, Array(String));
+ALTER TABLE `default`.`ratelimits_raw_v2` ADD COLUMN `override_id` String;
+ALTER TABLE `default`.`ratelimits_raw_v2` ADD COLUMN `limit` UInt64;
+ALTER TABLE `default`.`ratelimits_raw_v2` ADD COLUMN `remaining` UInt64;
+ALTER TABLE `default`.`ratelimits_raw_v2` ADD COLUMN `reset` Int64 CODEC(Delta(8), LZ4);
```

Comment:
my only nit is that this should be `reset_at` to eliminate any ambiguity as to whether this is a relative measurement or absolute timestamp